doc/SETUP.md

```diff
@@ -2,8 +2,21 @@
 
 ## 🚀 Quick Start
 
+- Install dependencies\
+  `$ yarn install`
+
 ### 💻 Load extension into browser
 
+- Start development build, which will automatically watch for file changes:
+
+  - Chrome\
+    `$ yarn run dev:chrome`
+  - Firefox\
+    `$ yarn run dev:firefox`
+  - Opera\
+     `$ yarn run dev:opera`
+    **NOTE:** by default, the extension built this way will talk to the development regtest API. In case you want to do manual tests against the production API, add the following `WALLET_CREATE_URL` environment variable to your command: `$ WALLET_CREATE_URL="https://getalby.com/api/users" yarn run dev:your-browser-of-choice`
```

Comment:
You wanna add an empty line here to separate it form the "opera" entry?
<img width="948" alt="image" src="https://user-images.githubusercontent.com/1016218/207873743-48c1f454-3520-40fb-988c-1b99ffa27016.png">

Reply:
thanks for spotting this. fixed here: https://github.com/getAlby/lightning-browser-extension/pull/1877#discussion_r1049653540